tests/testthat/_snaps/reporter-progress.md

```diff
@@ -564,7 +565,7 @@
     v | F W  S  OK | Context
     
     - [ FAIL 0 | WARN 0 | SKIP 0 | PASS 0 ] Starting up...                          
-    x | 11        0 | reporters/fail-many                                           
+    \ [ FAIL 11 | WARN 0 | SKIP 0 | PASS 0 ] @ reporters/fail-many                  
```

Comment:
This seems like a snapshot of the progress bar? Is it reproducible?

Reply:
Hmmm, the `\` at the beginning is a spinner, no? IDK why it would always stop in this position. The last part shows the test file currently running, I think that should not be there, either? Maybe we need to update / finish the progress bar?

Reply:
That is somewhat weird.... but.... yeah.